connectors-list.json

```diff
@@ -0,0 +1,214 @@
+[
+  {
+    "name": "@wrtnlabs/connector-arxiv-search",
+    "version": "0.0.1"
+  },
```

Comment:
The package's version follows the Connector's version(in root package.json).
so, we need to the code to modify the package version of monorepo in CI.
Or just add the Connector's version to `connectors-list.json`

Reply:
OK, so we can remove the package information of each packages